test/suite/test_timestamp01.py

```diff
@@ -37,6 +37,8 @@ def timestamp_str(t):
     return '%x' % t
 
 class test_timestamp01(wttest.WiredTigerTestCase, suite_subprocess):
+    session_config = 'isolation=snapshot'
+
```

Comment:
I think this test is the place to perhaps not make that change global in the test so that this test can add an error case of using timestamps without snapshot isolation.